helm/alfresco-content-services/values.yaml

```diff
@@ -68,7 +68,7 @@ transformrouter:
   replicaCount: 2
```

Comment:
@aymanthefirst Also the x5 T-Engines need to be bumped from 2.3.1 to 2.3.2 (to be consistent with docker-compose) ?

Reply:
I changed them all in this helm values file, and also in alfresco-bat/docker-compose.yaml https://github.com/Alfresco/alfresco-bat/pull/6/commits/ae4db73c1ef54166ca773e647d2364cf491163d1